website/pages/en/new-chain-integration.mdx

```diff
@@ -1,75 +1,150 @@
 ---
-title: Integrating New Networks
+title: New Chain Integration
```

Comment:
I noticed that the nav still calls this page "Integrating New Networks", and that's because it's been defined in `website/pages/en/_meta.js`. You should replace
```
'new-chain-integration': 'Integrating New Networks',
```
with
```
'new-chain-integration': '',
```